df.military.xml

```diff
@@ -160,7 +160,7 @@
             </pointer>
         </stl-vector>
 
-        <int32_t name="cur_alert_idx" refers-to='$$._parent.schedule[$]' init-value='0'/>
+        <int32_t name="cur_routine_idx" refers-to='$$._parent.schedule[$]' init-value='0'/>
```

Comment:
i've verified that there are no live C++ references to this field, but haven't checked scripts yet